baseplate/secrets/__init__.py

```diff
@@ -7,6 +7,7 @@
 
 from .store import (
     CorruptSecretError,
+    CredentialSecret,
```

Comment:
we probably don't need to explicitly export this. versioned secret is in here because of the `VersionedSecret.from_simple_secret` API but otherwise it won't be common for users to manually instantiate a `CredentialSecret` vs. getting one from `get_credentials()`